tests/platform_tests/test_reload_config.py

```diff
@@ -71,26 +71,27 @@ def test_reload_configuration_checks(duthosts, rand_one_dut_hostname, localhost,
 
     reboot(duthost, localhost, reboot_type="cold", wait=5)
     logging.info("Reload configuration check")
-    out = duthost.shell("sudo config reload -y", executable="/bin/bash")
+    out = duthost.shell("sudo config reload -y", executable="/bin/bash", module_ignore_errors=True)
     # config reload command shouldn't work immediately after system reboot
     assert "Retry later" in out['stdout']
+
     assert wait_until(300, 20, 0, config_system_checks_passed, duthost)
 
     # After the system checks succeed the config reload command should not throw error
-    out = duthost.shell("sudo config reload -y", executable="/bin/bash")
+    out = duthost.shell("sudo config reload -y", executable="/bin/bash", module_ignore_errors=True)
```

Comment:
This time we should not expect any error

Reply:
We do expected negative error when config reload fails after this PR https://github.com/Azure/sonic-utilities/pull/2200